### DISCUSSION TITLE
Logos and UI designs

### DISCUSSION THREAD

**Discussion by inorrr** [CSC207-2022F-UofT/course-project-macrosoft#57] [Ideas]
Is there any specific style/theme we should use? Reference below and all ideas are welcome! 

One possible approach: Minimalism design -> https://sympli.io/blog/the-art-of-minimalism-in-ui-design

Reasons:
1. Feasibility: well supported by Jframe design features.
2. Accessibility: Easy and clear to use.

**Comment by xuallenx** [comment]
I think our previous ideas of a blue black/ green black that were brought up in Saturdays meeting would look nice! I agree with the Minimalistic design, as this would allow us to implement more UI features easily in the future as well.

**Comment by jacksonsujh** [comment]
I think the idea of minimalistic design is awesome. One thing I may suggest is that we could use a colour that's more appetizing to the user.
